src/middlewares/errorHandler/errorHandler.middleware.ts

```diff
@@ -0,0 +1,12 @@
+import { Request, Response, NextFunction, ErrorRequestHandler } from 'express';
+
+const ErrorHandler: ErrorRequestHandler = (
+  err: Error,
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  res.status(500).send({ err, testing: 'testing!' });
```

Comment:
take out the testing lol

Reply:
lol I didn't even notice, thanks for pointing it out.